CITATION.cff

```diff
@@ -0,0 +1,71 @@
+cff-version: 1.2.0
+message: "Please cite the following when using this software: https://doi.org/10.1117/12.2629569"
+authors:
+- family-names: Jenness
```

Comment:
Should the software authors be only the paper authors?  That seems kind of restrictive.  But having all code contributors is hard to maintain.  I was going to use something like:
```
authors:
  - name: "Rubin Observatory Data Management Team"
```

Reply:
I was listing people who contributed to the software itself, not the paper authors. I can't find a CITATION.cff file in the astropy package so I'm not sure what others do for projects where it's more than a couple of contributors. Looking at the CITATION.cff docs it allows what you have and additional people so maybe I should add name as first author and then core team after that.

Reply:
maybe say "Rubin Observatory Data Management Team, including Tim J., Jim B., ....., and many other contributers"

Reply:
I think:

```yaml
authors:
  - name: "Rubin Observatory Data Management Team"
  - family-names: Jenness
    given-names: Tim
  - family-names: Bosch
    given-names: James
  ...
```
works within the standard.